### ISSUE TITLE
GGUF incompatibility

### ISSUE THREAD

**Issue by leedrake5** [ggerganov/llama.cpp#3285]
I am using the suggested oogabooga/text-generation-webui to load models and it works excellently, particularly with the llama-cpp-python package. However I am starting to get an unexpected error when loading models that are less than 3 weeks old: 

`Exception: You are using an outdated GGUF, please download a new one.
`

Is there any way to identify and/or convert GGUF files that are made earlier than current? Just based on the file, it is hard to ascertain which are supported and which are not. I know this is primarily an issue with text-generation-webui or llama-cpp-python, but it seems like convert.py could be a way to quickly re-authenticate gguf files as development progresses. 

It is also frustrating to have to re-download every few weeks based on updates - many internet service providers have data caps. In addition, it may be difficult to find, or identify, what version of a GGUF is compatible, with the current or past versions of llama.cpp

Long story short, is there any plan for how to address ghost obsolete gguf files?

**Comment by slaren** [comment]
There are two versions of GGUF, and llama.cpp is capable of loading both of them. That error is from the [GGUF metadata parser](https://github.com/oobabooga/text-generation-webui/blob/029da9563f5fc8602dd03724044526f09849b00b/modules/metadata_gguf.py) of oogabooga/text-generation-webui. You would have to ask them to add support for GGUFv1. Alternatively, you can use the `quantize` tool from this repository with `COPY` mode to update your model to GGUFv2.
